package com.redhat.ceylon.compiler.test.statement.trycatch;

class ThrowExceptionNamedArgs {
    
    private final void t() {
        throw new .ceylon.language.NamedArgumentCall<java.lang.Void>(null, .ceylon.language.String.instance("Bang!"), null){
            
            public .ceylon.language.Exception call$() {
                return new .ceylon.language.Exception((ceylon.language.String)this.args[0], (.java.lang.Throwable)this.args[1]);
            }
        }.call$();
    }
    
    ThrowExceptionNamedArgs() {
    }
    
    public static void main(.java.lang.String[] args) {
        ceylon.language.process.getProcess().setupArguments(args);
        new ThrowExceptionNamedArgs();
    }
}